### ISSUE TITLE
HTTPS/SSL Grading shown with Encrypted Connection

### ISSUE THREAD

**Issue by Lionirdeadman** [duckduckgo/duckduckgo-privacy-extension#106]
I think the extension should show this to make sure it's not simply weak.

This optionally could affect the privacy grade.

**Comment by alohaas** [comment]
@Lionirdeadman I'm not 100% clear on your feedback. Encryption currently factors into the Privacy Grade for each site. Are you asking for a more nuanced report on the connection and certificate? Please provide more details if possible.

**Comment by Lionirdeadman** [comment]
Yeah, I think it should be more nuanced by taking into account the cipher, certificate and other things.

Similar to this : https://www.ssllabs.com/ssltest/

**Comment by alohaas** [comment]
Gotcha. Thanks for explaining.
